### PR TITLE
Add Support for route_icon

### PR DESCRIPTION
**Pull request by consindo**
-  If you want to add an icon, you'll need to have two - the color variant and the mono variant, prefixed accordingly.
- There's a SkyBus icon, but it's disabled unless they give us lots of money

### CODE REVIEW

**Comment by darth10** [comment]
bitmoji

File: server-worker/lines/index.js [hunk 1]
```diff
@@ -111,6 +122,7 @@ var line = {
         longName: cityMetadata[global.config.prefix].longName,
       },
       colors: lineData.lineColors || {},
+      icons: lineData.lineIcons || {},
```
Comment (mjdavidson) [inline server-worker/lines/index.js]: i really don't like this way of doing defaults

File: server-worker/lines/index.js [hunk 2]
```diff
@@ -29,9 +29,16 @@ var line = {
     if (lineData.getColor) {
       return lineData.getColor(agency_id, route_short_name)
     } else if (lineData.lineColors) {
-      return lineData.lineColors[route_short_name] || '#000'
+      return lineData.lineColors[route_short_name] || '#00263A'
     }
-    return '#000'
+    return '#00263A'
+  },
+  getIcon: function(agency_id, route_short_name) {
```
Comment (mjdavidson) [inline server-worker/lines/index.js]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions#Description

File: server-worker/lines/regions/nz-akl.js [hunk 3]
```diff
@@ -1046,6 +1054,7 @@ cache.ready.push(cacheOperatorsAndShapes)
 
 module.exports = {
   lineColors: lineColors,
+  lineIcons: lineIcons,
```
Comment (mjdavidson) [inline server-worker/lines/regions/nz-akl.js]: can't this be shortened to { lineColors, lineIcons, ...rest }

File: server-worker/lines/index.js (hunk 2 above)
Comment (consindo) [reply server-worker/lines/index.js]: embarassed I didn't know this

File: server-worker/lines/index.js (hunk 1 above)
Comment (consindo) [reply server-worker/lines/index.js]: okay, will fix

File: server-worker/lines/regions/nz-akl.js (hunk 3 above)
Comment (consindo) [reply server-worker/lines/regions/nz-akl.js]: yes - will fix

**Pull request merged**